docs/quickstarts/packet.md

```diff
@@ -5,262 +5,311 @@
-* `kubectl` installed locally to access the Kubernetes cluster.
+The guide uses [Amazon Route 53](https://aws.amazon.com/route53/) as a DNS provider. For more
+information on how Lokomotive handles DNS, refer to [this](../concepts/dns.md) document.
 
-## Steps
+[Lokomotive components](../concepts/components.md) complement the "stock" Kubernetes functionality
+by adding features such as load balancing, persistent storage and monitoring to a cluster. To keep
+this guide short you will deploy a single component - `httpbin` - which serves as a demo
+application to verify the cluster behaves as expected.
 
-### Step 1: Install lokoctl
+## Requirements
 
-lokoctl is a command-line interface for Lokomotive.
+* A Packet account with a project created and
+  [local BGP](https://www.packet.com/developers/docs/network/advanced/local-and-global-bgp/)
+  enabled.
+* A Packet project ID.
+* A Packet
+  [user level API key](https://www.packet.com/developers/docs/API/getting-started/)
+  with access to the relevant project.
+* An AWS account.
+* An AWS
+  [access key ID and secret](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
+  of a user with
+  [permissions](https://github.com/kinvolk/lokomotive/blob/master/docs/concepts/dns.md#aws-route-53)
+  to edit Route 53 records.
+* An AWS Route 53 zone (can be a subdomain).
+* An SSH key pair for accessing the cluster nodes.
+* Terraform `v0.12.x`
+  [installed](https://learn.hashicorp.com/terraform/getting-started/install.html#install-terraform).
+* The `ct` Terraform provider `v0.5.0`
+  [installed](https://github.com/poseidon/terraform-provider-ct#install).
+* `kubectl` [installed](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+>NOTE: The `kubectl` version used to interact with a Kubernetes cluster needs to be compatible with
+>the version of the Kubernetes control plane. Ideally you should install a `kubectl` binary whose
+>version is identical to the Kubernetes control plane included with a Lokomotive release. However,
+>some degree of version "skew" is tolerated - see the Kubernetes
+>[version skew policy](https://kubernetes.io/docs/setup/release/version-skew-policy/) document for
+>more information. You can determine the version of the Kubernetes control plane included with a
+>Lokomotive release by looking at the [release notes][releases].
 
-To install `lokoctl`, follow the instructions in the [lokoctl installation](../installer/lokoctl.md)
-guide.
+## Steps
 
-### Step 2: Set up a working directory
+### Step 1: Install lokoctl
 
-It's better to start fresh in a new working directory, as the state of the cluster is stored in this
-directory.
+`lokoctl` is the command-line interface for managing Lokomotive clusters.
 
-This also makes the cleanup task easier.
+Download the latest `lokoctl` binary for your platform:
 
 ```console
-mkdir -p lokomotive-infra/mycluster
-cd lokomotive-infra/mycluster
-```
+export os=linux  # Linux
```

Comment:
Do we want to default to linux and add in the comment that `darwin` is also possible? So users copying and pasting get the linux version by default?

If you think it's clear for the user that they should copy one of the lines (linux or darwing) feel free to ignore.

Reply:
Done.